src/app/features/dashboard/dashboard.component.ts

```diff
@@ -113,6 +118,18 @@ export class DashboardComponent implements OnInit, AfterViewInit, OnDestroy {
       map((token) => token!.roles),
       map((roles) => roles.map((role) => ({ routerLink: [`/arpa/dashboard/${role}`], label: role.toUpperCase() })))
     );
+
+    this.meService.getProfilesMusician().subscribe((profile: MyMusicianProfileDto | MyMusicianProfileDto[]) => {
+      let musicianProfile: MyMusicianProfileDto | null;
+
+      if (Array.isArray(profile)) {
+        musicianProfile = profile.length > 0 ? profile[0] : null;
+      } else {
+        musicianProfile = profile;
+      }
+
+      this.hasMusicianProfile = musicianProfile !== null && musicianProfile.id != null;
+    });
```

Comment:
du könntest this.hasMusicianProfile zu einem Observable<boolean> machen, im html mit | async abrufen und dir dafür hier das .subscribe sparen. statdessen mit .pipe(map auf das boolean observable mappen